src/OrchardCore.Themes/TheAdmin/Assets/js/admin.js

```diff
@@ -193,3 +193,28 @@ $('body').on('submit', 'form.no-multisubmit', function (e) {
         form.removeClass(submittingClass);
     }, 5000);
 });
+
+// Check all checkboxes
+$(function () {
```

Comment:
jQuery 🥵

Reply:
I just copied code that was already there :wink:

Reply:
Yes I know. But I am sure you can re-write it :). Anyway up to you. But I think it should live it a separate file. And for the new resource, it has to depend on jquery since jquery is not loaded by default.

Reply:
Nah, I rewrote it.

Reply:
Sweet!